csfieldguide/static/scss/website.scss

```diff
@@ -302,3 +303,26 @@ a {
     color: #212529;
   }
 }
+
+// Standard bootstrap-defined Medium devices boundary
+@media (max-width: 767px) {
+  .navbar-collapse {
+    background: #2a3da0;
```

Comment:
This seems like a bug, but let's leave this fix in for the moment, and check again once we update Bootstrap.